README.md

```diff
@@ -129,6 +131,19 @@ Solomon proposed to support VXLAN from the Docker core.
 - Docker Hub Enterprise
 - Private Docker registries using the “registry" docker image.
 
+## Configuration Management Tools with Docker support 
+
+- Ansible 
+- Saltstack
+- Chef 
+- Puppet 
+- Packer
+
+## Networking with Docker
+
+- [Skydock](https://github.com/crosbymichael/skydock) 
+- Vulcand 
```

Comment:
Valcand is a reverse proxy. My understanding is that neither Valcand as a whole nor its subsystems were specifically designed or optimized for Docker. Although it uses etcd which is commonly used in the Docker community, my preference is to omit it from this survey.